documentation/ame-guide/modules/ROOT/pages/namespace/use-references.adoc

```diff
@@ -17,23 +17,19 @@ To add references, proceed as follows:
 +
 image::external-reference-drag-and-drop.png[width=65%]
 +
-[NOTE]
-====
-References can only be dragged once into the current Aspect Model. These elements are marked with a special icon signifying that they are from a different location. You can only edit an reference by editing the file it is defined in. By double clicking an reference the file where that element exists will be opened in a different window
+→ A reference has now been added to your model.
 
-image::external-reference.png[width=80%]
-====
-+
 [NOTE]
 ====
-When you drag and drop an reference, it will not include all the lower connections (dependencies) of the element.
+* References can only be dragged once into the current Aspect Model. These elements are marked with a special icon signifying that they are from a different location. You can only edit a reference by editing the file it is defined in. By double clicking a reference the file where that element exists will be opened in a different window.
```

Comment:
You can only edit a reference by editing the file it is defined in. --> rephrase? (easier)
double-clicking?